Mirage-Widget/Assets/MirageWidget/Examples/MirageWidgetExample.cs

```diff
@@ -18,5 +18,13 @@ private void Start()
 			_messageBus.AddListener(_mirageWebViewManager);
 			_mirageWebViewManager.ShowWebView("https://google.com");
 		}
+		
+		private void OnApplicationPause(bool pauseStatus)
+		{
+			if (pauseStatus)
+			{
+				Application.targetFrameRate = 30;
```

Comment:
Our users will not like that we are playing with their framerate settings. Why do we need that? If this is required we should set the value back. This is a really strong implicit effect

Reply:
You are right, this is redundant.